tests/main/snap-quota-journal/task.yaml

```diff
@@ -0,0 +1,100 @@
+summary: Functional test for journal quota-related snap commands.
+
+details: |
+  Functional test for snap journal quota group commands ensuring that they are 
+  effective in practice.
+
+# these systems do not support journal quota groups due to their old systemd versions.
+# requires systemd v245+
+systems:
+  - -amazon-linux-2-*
+  - -centos-7-*
+  - -centos-8-*
+  - -debian-10-64
+  - -ubuntu-14.04-*
+  - -ubuntu-16.04-*
+  - -ubuntu-18.04-*
+  - -ubuntu-core-16-*
+  - -ubuntu-core-18-*
+
+prepare: |
+  snap pack test-snapd-journal-quota
```

Comment:
here you can use -> $TESTSTOOLS"/snaps-state install-local test-snapd-journal-quota

Reply:
Updated to this approach, thanks for the review!